src/App.jsx

```diff
@@ -14,33 +15,81 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const copyGoods = [...goodsFromServer];
+  const [sortType, setSort] = useState('');
+  const [isReversed, setReversed] = useState(false);
```

Comment:
Renaming here will make the names more clear and consistent.

```suggestion
  const [sortOrder, setSortOrder] = useState('');
  const [isReversed, setIsReversed] = useState(false);
```